src/App.js

```diff
@@ -1,22 +1,67 @@
 import logo from './logo.svg';
 import './App.css';
+import React from "react";
+import { THEMES, setTheme } from "./theme.js";
 
 function App() {
```

Comment:
For modularity and keeping the initial file as high-level as possible, this content should be extracted to a new component.